gprofiler/profilers/java.py

```diff
@@ -273,11 +302,91 @@ def make_application_metadata(self, process: Process) -> Dict[str, Any]:
         # that loads other libs.
         libjvm_elfid = get_mapped_dso_elf_id(process, "/libjvm")
 
-        metadata = {"java_version": version, "libjvm_elfid": libjvm_elfid}
+        try:
+            jvm_flags = self.get_jvm_flags_serialized(process)
+        except CalledProcessError as e:
+            jvm_flags = [{"error": e.stderr}]
```

Comment:
`e.stderr` can have high cardinality, let's just use the same `type(e).__name__` & log the exception? (i.e just remove this except clause and fallthrough the default one). We wil be able to check the logs for exact details of the failure.

Reply:
The exception is already logged in the ibber function that rethrows the exception, I'll add another log here for the general case.

btw e.stderr includes pid or something?

Reply:
Run this code and see the logs emitted by `run_process` - they include the full commandline, so yeah.